vendor/k8s.io/kubernetes/cmd/kubelet/app/patch_volumequota.go

```diff
@@ -83,6 +83,10 @@ func PatchVolumePluginsForLocalQuota(rootdir string, plugins *[]volume.VolumePlu
 		return fmt.Errorf("unable to parse \"%s\" as a quantity", volumeConfig.LocalQuota.PerFSGroup)
 	}
 
+	if quota.Value() <= 0 {
```

Comment:
Is perFSGroup able to be set to `0`? Then this comparison would be < 0. Just double checking

Reply:
Basically that would make all of the emptydirs effectively read-only and.. well.. permanently empty.  So that didn't seem like something someone would ever want in my mind.  Granted, you could set it to 1 bytes and probably be in the same boat but if you set a non-zero lower limit, you'd have to document it and I didn't want to do that :smile:

Reply:
Makes sense, thanks!